pkg/api/aim/dashboards.go

```diff
@@ -106,13 +112,19 @@ func GetDashboard(c *fiber.Ctx) error {
 		},
 	}
 	if err := database.DB.
+		InnerJoins(
+			"App",
+			database.DB.Select(
+				"ID", "Type",
+			).Where(
+				&database.App{NamespaceID: ns.ID}, "NamespaceID",
+			),
+		).
 		Where("NOT dashboards.is_archived").
-		Preload("App").
 		Joins("LEFT JOIN apps ON apps.id = dashboards.app_id").
```

Comment:
This `Joins` needs to be removed

Reply:
nope. this is wasn't a problem unfortunately.